File: src/main/java/org/funivan/lologa/algo/gameplay/GameplayInterface.java
package org.funivan.lologa.algo.gameplay;

import org.funivan.lologa.tile.TileInterface;
import org.funivan.lologa.tiles.TilesInterface;

public interface GameplayInterface {
    int minimum();

    TilesInterface interact(TileInterface tile, TilesInterface tiles);
}

File: src/main/java/org/funivan/lologa/algo/gameplay/GameplayInterface.kt
package org.funivan.lologa.algo.gameplay

import org.funivan.lologa.tile.TileInterface
import org.funivan.lologa.tiles.TilesInterface

interface GameplayInterface {
    fun minimum(): Int

    fun interact(tile: TileInterface, tiles: TilesInterface): TilesInterface
}
